### PR TITLE
Updated Week 3 README.md

### PR DESCRIPTION
**Pull request by uday-rana**
Removed a `,`

### CODE REVIEW

File: weeks/week03/README.md [hunk 1]
```diff
@@ -406,7 +406,7 @@ and define *how many* times a pattern or match can/must occur.  We do this by ad
 
 * `+` is similar to `*` but means *one or more*. `vroo+m` would match `"vroom"` but also `"vroooooooom"` and `"vroooooooooooooooooooooooooooooooom"`
 
-* We can limit the number of matches to an exact number using `{n}`, which means *match exactly `n` times*. `vroo{3}m` would only match `"vroooom"`.  We can further specify that we want a match to happen  *match `n` or more times* using `{n,}`, or use `{n,m}` to indicate we want to match *at least `n` times and no more than `m` times: `\w{8,16}` would match 8 to 16 word characters, `"ABCD1234"` or `"zA5YncUI24T_3GHO"`
+* We can limit the number of matches to an exact number using `{n}`, which means *match exactly `n` times*. `vroo{3}m` would only match `"vroooom"`.  We can further specify that we want a match to happen  *match `n` or more times* using `{n}`, or use `{n,m}` to indicate we want to match *at least `n` times and no more than `m` times: `\w{8,16}` would match 8 to 16 word characters, `"ABCD1234"` or `"zA5YncUI24T_3GHO"`
```
Comment (humphd) [inline weeks/week03/README.md]: This comma has significance:

- `{5}` means exactly five
- `{5,}` means five or more
- `{5,8}` means five, six, seven, or eight

Comment (uday-rana) [reply weeks/week03/README.md]: I see, I misunderstood. Thank you

File: weeks/week03/README.md [hunk 2]
```diff
@@ -473,4 +473,4 @@ Write a series of functions to accomplish the following, building a larger progr
 6. After doing all of the above steps, create a new record with `ID`, `Name`, `Area Code`, `Height In Inches` and separate them with commas
 7. Combine all these processed records into a new CSV formatted string, with rows separated by `\n`.
 
-A sample solution is provided in [solution.js](solution.js).
\ No newline at end of file
+A sample solution is provided in [solution.js](solution.js).
```
Comment (humphd) [inline weeks/week03/README.md:476]: Do you want to fix this PR to remove the `,` change above, but leave this one?  Then I can merge it.

Comment (uday-rana) [reply weeks/week03/README.md:476]: Done!

**Review by humphd — APPROVED** [review]

**Comment by humphd** [comment]
@uday-rana Reading this PR again, nothing has changed from the original, so maybe it's not needed?  Let me know if I'm missing something.  I appreciate you contributing and trying to improve the docs, it's great.

**Comment by uday-rana** [comment]
I only made the `,` change, but when I did, it showed the last line as having changed too, which you then approved, but looks like you're right. Not sure why it shows up as a change in the first place.

**Comment by humphd** [comment]
OK, make sense.  I'll close this then, but send more if you find more problems!

**Pull request closed without merging**